yarn/examples/hello_world/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "hello_world",
+  "version": "0.0.1",
+  "description": "Expressjs hello world",
+  "main": "index.js",
```

Comment:
For completeness it would be a good idea to include an `engines` field that specifies a node version that supports arrow syntax since it is being used in the tests.